glusterd2/commands/volumes/volume-options-get.go

```diff
@@ -0,0 +1,99 @@
+package volumecommands
+
+import (
+	"net/http"
+
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/glusterd2/xlator"
+	"github.com/gluster/glusterd2/glusterd2/xlator/options"
+	"github.com/gluster/glusterd2/pkg/api"
+
+	"github.com/gorilla/mux"
+)
+
+func volumeOptionsGetHandler(w http.ResponseWriter, r *http.Request) {
+
+	ctx := r.Context()
+
+	volname := mux.Vars(r)["volname"]
+
+	volinfo, err := volume.GetVolume(volname)
+	if err != nil {
+		status, err := restutils.ErrToStatusCode(err)
+		restutils.SendHTTPError(ctx, w, status, err)
+		return
+	}
+
+	optname, found := mux.Vars(r)["optname"]
+
+	if found {
+		opt, err := xlator.FindOption(optname)
+		if err != nil {
+			restutils.SendHTTPError(ctx, w, http.StatusNotFound, err)
+			return
+		}
+		resp := createVolumeOptionGetResp(volinfo, opt)
+		restutils.SendHTTPResponse(ctx, w, http.StatusOK, resp)
+	} else {
+		resp := createVolumeOptionsGetResp(volinfo)
+		restutils.SendHTTPResponse(ctx, w, http.StatusOK, resp)
+	}
+}
+
+func createVolumeOptionGetResp(volinfo *volume.Volinfo, opt *options.Option) *api.VolumeOptionGetResp {
+	var resp api.VolumeOptionGetResp
+
+	for _, xl := range xlator.Xlators() {
+		modified, found := false, false
+		val := opt.DefaultValue
+
+		for _, k := range opt.Key {
+			if val, found = volinfo.Options[xl.ID+"."+k]; found {
+				modified = found
```

Comment:
Instead of setting it to value of `found`, although the value is true, it's easier to understand if you explicitly set it to `true`

Reply:
done